src/fable-js/fable-core.js

```diff
@@ -315,7 +315,9 @@
           case "f": case "F": rep = rep.toFixed(precision || 6); break;
           case "g": case "G": rep = rep.toPrecision(precision); break;
           case "e": case "E": rep = rep.toExponential(precision); break;
-          case "A": rep = JSON.stringify(rep); break;
+          case "A": rep = JSON.stringify(rep, function(k, v) {
+                        return ArrayBuffer.isView(v) ? Array.from(v) : v;
```

Comment:
It's difficult to make `%A` work perfectly, but this is a very nice addition :) May I suggest to include other iterables too (lists, sets, maps)? Something like
```
return v && v[Symbol.iterator] && !Array.isArray(v) ? Array.from(v) : v;
```